web/ci.sh

```diff
@@ -24,15 +24,22 @@ cd "$THIS_SCRIPT_PATH"
 
 builder_describe "Defines and implements the CI build steps for Keyman Engine for Web (KMW)." \
   "build" \
-  "test             Runs all unit tests."  \
-  "post-test        Runs post-test cleanup.  Should be run even if a prior step fails." \
-  "validate-size    Runs the build-size comparison check" \
-  "--debug          Runs this script in local-development mode; reports and tests will be locally logged"
+  "test                 Runs all unit tests."  \
+  "post-test            Runs post-test cleanup.  Should be run even if a prior step fails." \
+  "validate-size        Runs the build-size comparison check" \
+  "publish-s.keyman     Prepares an s.keyman.com PR (intended for release builds)" \
```

Comment:
I assume you'll be fine to merge in favour of #8001?

Reply:
> Commit history may take a tumble here due to all the rebases; I originally did a merge of the prior PRs in this set with the 📜 set before starting the current branch.

Now that the feature branch has had recent changes merged into it, as well as the preceding PRs in the chain... looks like there's a merge conflict to resolve there.  Probably nothing too crazy though, and yes - will give priority to #8001's changeset.  This PR's version is that of an older state of development, after all.